bin/good-first-issue.js

```diff
@@ -3,18 +3,20 @@
 var cli = require('commander')
 var chalk = require('chalk')
 var opn = require('open')
+const gfi = require('libgfi')
 
-var pJson = require('../package.json')
-
+var packageJSON = require('../package.json')
 const log = require('../lib/log')
+const prompt = require('../lib/prompt')
 const projects = require('../data/projects.json')
-const goodFirstIssue = require('..')
 
-const prompt = require('./prompt')
+const options = { // options for libgfi
+  projects: projects
```

Comment:
Can simplify this bit:

```suggestion
  projects
```